UNRELEASED.md

```diff
@@ -15,6 +15,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Bug fixes
 
 - Fixed alignment of `Page` and `TopBar` so the search aligns to the page. ([#3610](https://github.com/Shopify/polaris-react/pull/3610))
+- Fixed `FocusManager` adding inactive items([#3630](https://github.com/Shopify/polaris-react/pull/3630))
```

Comment:
Can you elaborate on this? It's hard to tell exactly what the fix is here

Reply:
```suggestion
- Fixed `FocusManager` from tracking inactive items that prevented trap focusing([#3630](https://github.com/Shopify/polaris-react/pull/3630))
```